src/cmd/create.go

```diff
@@ -710,18 +710,42 @@ func pullImage(image, release, authFile string) (bool, error) {
 		shouldPullImage = true
 	}
 
+	imageSizeChan := make(chan string)
+
 	if promptForDownload {
 		fmt.Println("Image required to create toolbox container.")
+		// Launch a goroutine to call getImageSizeFromRegistry and send the size to the channel
+		go func() {
+			imageSize, err := getImageSizeFromRegistry(imageFull)
+			if err != nil {
+				logrus.Debugf("Getting image size failed: %s", err)
```

Comment:
We should send the `error` back over a channel and have the main goroutine do the logging.  Otherwise, the logs will interfere with the confirmation prompt.